server/users/routes.ts

```diff
@@ -376,6 +376,7 @@ router.patch("/@me/settings", ensureLoggedIn, async (req, res) => {
 
     res.status(200).json(aggregateByType(settings));
   } catch (e) {
+    // TODO: there are three different awaits in the try block, only one whose failure would suggest the settings weren't updated, consider adjusting this message
```

Comment:
This is a question I had; didn't want to update this message without checking - can this be changed to something more general like "Something went wrong while updating settings?" At a glance it looks possible that settings might update but then something goes wrong in the subsequent tasks.

Reply:
You're very much right, and it might make sense to just wrap the first `await` in the try catch with the custom message. I'm 95% sure if we get an error in the subsequent one, the server will automatically generate a 500 error anyway.